From 49fe650c8651fcc6bb13ab854efe3b329de6d33b Mon Sep 17 00:00:00 2001
From: Antheas Kapenekakis <lkml@antheas.dev>
Date: Wed, 20 Aug 2025 16:49:11 +0200
Subject: [PATCH 091/112] platform/x86: ayaneo-ec: Remove charge inhibition for
 other devices

On Bazzite, charge inhibition is currently provided by the oot
ayaneo-platform driver. To avoid stacking modules, remove the support
from the ayaneo-ec driver.

Signed-off-by: Antheas Kapenekakis <lkml@antheas.dev>
---
 drivers/platform/x86/ayaneo-ec.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/platform/x86/ayaneo-ec.c b/drivers/platform/x86/ayaneo-ec.c
index 41a24e091..b7fc7ee44 100644
--- a/drivers/platform/x86/ayaneo-ec.c
+++ b/drivers/platform/x86/ayaneo-ec.c
@@ -66,7 +66,6 @@ static const struct ayaneo_ec_quirk quirk_fan = {
 
 static const struct ayaneo_ec_quirk quirk_charge_limit = {
 	.has_fan_control = true,
-	.has_charge_control = true,
 };
 
 static const struct ayaneo_ec_quirk quirk_ayaneo3 = {
-- 
2.52.0

